app/models/item.rb

```diff
@@ -8,4 +8,12 @@ class Item < ApplicationRecord
   validates_numericality_of :unit_price, only_integer: true
 
   enum status: [:disabled, :enabled]
+  
+  
+  
+  def invoice_date
+    invoice_id = InvoiceItem.where(item_id:self.id).pluck(:invoice_id).first
+    
+    Invoice.where(id: invoice_id).first.created_at.strftime("%A, %B %d, %Y")
```

Comment:
i'd argue that putting the `.strftime` part of this method in the View adheres more to the MVC principle. Worth the conversation I think at leas...